### DISCUSSION TITLE
FR: Sort the labels by color

### DISCUSSION THREAD

**Discussion by mesqueeb** [community/community#13065] [Projects and Issues]
Many of us, use Github Labels with some sort of color coding. (just like the official [Github Roadmap](https://github.com/orgs/github/projects/4247))

It would be very handy if we can **have our labels sorted by color** on the `/labels` page of a repo.

Sorting by color would be an addition to these current possibilities:

<img width="518" alt="image" src="https://user-images.githubusercontent.com/3253920/159143354-495a4066-50b5-4573-b254-31885b029033.png">

**Comment by Yuri05** [comment]
This feature wold be indeed very helpful

**Comment by jfullstackdev** [comment]
how would you sort that by color, black to white, white to black etc. ? 

but there is already the Label filter with those colors associated 

![image](https://user-images.githubusercontent.com/47092464/182186492-37356478-f7f5-4058-b4c0-018de5709927.png)

so the colors will help identifying the issues at first look

**Reply by Yuri05** [reply]
As stated in the original post, it's about sorting on the **labels page of a repo** (via adding additional sort option "by colour").
And sorting by the colour hex code should not be too complicated(?)

**Reply by jfullstackdev** [reply]
yes I'm getting that, but what will be the great of importance sorting by color ? it's already color sorted at the Labels filter to help us identifying issues easily through color, 

when you select only the `bug` label, we all know it's sorted by red and when viewing it randomly without filter, it really helps us
